tests/Scout/ScoutIntegrationTest.php

```diff
@@ -38,6 +39,9 @@ protected function getEnvironmentSetUp($app): void
 
         $app['config']->set('scout.driver', 'mongodb');
         $app['config']->set('scout.prefix', 'prefix_');
+        $app['config']->set('scout.mongodb.index-definitions', [
+            'prefix_scout_users' => ['mappings' => ['dynamic' => true, 'fields' => ['bool_field' => ['type' => 'boolean']]]],
```

Comment:
Is it sensible to combine `dynamic: true` with explicit field mappings? I don't recall seeing anything like this in the server manual or Atlas docs.